### PR TITLE
Updating main.py to update on call slack group

### PR DESCRIPTION
**Pull request by mikegardner2087**
https://github.com/dapperlabs/SRE/issues/3008

Updating main.py to include;
Get on call user email from pagerduty
Get slack ID of user on call
add user to on call Slack group

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -42,23 +42,12 @@ def get_user(schedule_id):
     payload['until'] = now.isoformat()
     normal = get(normal_url, headers=headers, params=payload)
     if normal.status_code == 404:
-        logger.critical(f"ABORT: Not a valid schedule: {schedule_id}")
+        #logger.critical(f"ABORT: Not a valid schedule: {schedule_id}")
```
Comment (mtmendonca) [inline main.py]: Why is this commented out?

File: main.py [hunk 2]
```diff
@@ -147,4 +168,4 @@ def handler(request, event):
 
 
 if __name__ == "__main__":
-    handler(None, None)
+    handler(None, None)
```
Comment (mtmendonca) [inline main.py]:
```suggestion
    handler(None, None)

```

Comment (mikegardner2087) [reply main.py]: Don't get the suggestion? An extra blank line?

File: main.py (hunk 1 above)
Comment (mikegardner2087) [reply main.py]: Was when I was testing accidentally left it commented out

**Pull request closed without merging**